pywbem_mock/_inmemoryrepository.py

```diff
@@ -0,0 +1,279 @@
+#
+# (C) Copyright 2020 InovaDevelopment.comn
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+# Author: Karl  Schopmeyer <inovadevelopment.com>
+#
+
+"""
+The pywbem_mock module provides a standard interface to a CIM repository that
+stores collections of :class:`~pywbem.CIMClass`, :class:`~pywbem.CIMInstance`,
+and :class:`~pywbem.CIMQualifierDeclaration` as Python asbstract classes in the
+module :py:mod:`pywbem_mock/_baserepository.py`.
+
+The pywbem_mock inmemory CIM repository implements a repository derived from
+:py:mod:`pywbem_mock/_baserepository.py` that stores the CIM objects only in
+memory. This CIMrepository is created each time a
+:class:`~pywbem_mock.FakedWBEMConnection` is constructed and destroyed each
+time it is destroyed.
```

Comment:
Several issues in this paragraph:
* "The [...] repository implements a repository": Why not mention the InMemoryRepository class as a link?
* Again, remove mentioning of the module defining the base classes.
* Missing space in "CIMrepository".

Reply:
Modified.  Note that I modified second paragraph and added comment  about namespaces